foundations/javascript_basics/clean_code.md

```diff
@@ -148,7 +148,7 @@ You might wonder why this variable is declared with all caps when we recommended
 
 ### Indentation and line length
 
-Now it's time to head to more controversial topics. The war between coders that use tabs and coders that use spaces to indent their code is essentially a joke by now, as demonstrated in the [tabs versus spaces scene from the show Silicon Valley](https://www.youtube.com/watch?v=SsoOG6ZeyUI).
+Now it's time to head to more controversial topics. The war between coders that use tabs and coders that use spaces to indent their code is essentially a [joke](https://www.reddit.com/r/programming/comments/p1j1c/tabs_vs_spaces_vs_both/) by now.
```

Comment:
Rephrase suggestion to allow for a more accessible link text label

```suggestion
Now it's time to head to more controversial topics. There's already a joke about the [war between coders who indent with spaces versus tabs](https://www.reddit.com/r/programming/comments/p1j1c/tabs_vs_spaces_vs_both/).
```

Reply:
I find this wording "There's a joke about" to have a different meaning than "essentially being a joke".

I would change it to "The [war betwen ... with spaces versus tabs](link) is essentially a joke."

Reply:
My reading of the original "essentially a joke" wasn't mocking the spaces/tabs war, but that the situation literally is a joke in the community, which the suggested change maintains the meaning of.